server/plugin/plugin.go

```diff
@@ -131,5 +221,8 @@ func (p *MatterpollPlugin) SendEphemeralPost(channelID, userID, message string)
 	ephemeralPost.AddProp("override_username", responseUsername)
 	ephemeralPost.AddProp("override_icon_url", fmt.Sprintf(responseIconURL, *p.ServerConfig.ServiceSettings.SiteURL, PluginId))
 	ephemeralPost.AddProp("from_webhook", "true")
-	_ = p.API.SendEphemeralPost(userID, ephemeralPost)
+	err := p.API.SendEphemeralPost(userID, ephemeralPost)
```

Comment:
`SendEphemeralPost` returns `*model.Post`, so this error check isn't needed.
https://developers.mattermost.com/extend/plugins/server/reference/#API.SendEphemeralPost

Sorry, I missed this.

Reply:
Fixed